test/integration-test-main.sh

```diff
@@ -651,6 +642,7 @@ function test_multipart_copy {
 function test_multipart_mix {
     describe "Testing multi-part mix ..."
 
+    # TODO: why is this necessary?
```

Comment:
I think I decided to clear the cache file here to avoid reaching the disk capacity limit when executing Github Actions on macos.